### PR TITLE
Update Dutch translation

### PR DESCRIPTION
**Pull request by dennisschagt**
Before changing to non-draft PR and requesting review:
- [x] Rebase onto master after PR #738 is merged
- [x] Verify I've not made any obvious mistakes

### CODE REVIEW

File: po/nl.po [hunk 1]
```diff
@@ -6,14 +6,14 @@ msgstr ""
 "Project-Id-Version: newsboat 2.7\n"
 "Report-Msgid-Bugs-To: https://github.com/newsboat/newsboat/issues\n"
 "POT-Creation-Date: 2020-02-02 16:53+0300\n"
-"PO-Revision-Date: 2013-10-26 22:20+0100\n"
+"PO-Revision-Date: 2020-02-04 22:01+0100\n"
 "Last-Translator: Erwin Poeze <donnut@outlook.com>\n"
 "Language-Team: Dutch <vertalen@vrijschrift.org>\n"
```
Comment (Minoru) [inline po/nl.po]: You should put your name into `Last-Translator` and `Language-Team` fields, and move Erwin Poeze into `Language-Team` as well. (We don't really have translation teams in Newsboat, so the convention is to list every translator in the team field.)

**Comment by Minoru** [comment]
Thank you very much for this! Translations need very specific skills (i.e. knowledge of English and target language), making their upkeep very hard when there aren't many contributors.

BTW, #738 (and all my other pull requests) are *really* free for everyone to review. I challenge you to find at least one nitpick in my code :P

**Comment by dennisschagt** [comment]
> BTW, #738 (and all my other pull requests) are _really_ free for everyone to review. I challenge you to find at least one nitpick in my code :P

Challenge accepted

**Comment by Minoru** [comment]
#738 is merged, so this can be rebased onto master now.

**Comment by Minoru** [comment]
Thanks!

**Pull request merged**